2016/06/03 19:30:48 [error] 2926#0: *6 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/03 19:30:52 [error] 2926#0: *8 open() "/home/pi/pyprojects/portfolio_django_1.9/static/images/favicon.ico" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/images/favicon.ico HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/03 19:38:00 [error] 2926#0: *11 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/en/"
2016/06/03 19:38:07 [error] 2926#0: *9 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/fr/"
2016/06/03 19:44:32 [crit] 645#0: *1 connect() to unix:/home/pi/pyprojects/portfolio_django_1.9/run/gunicorn.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /fr/ HTTP/1.1", upstream: "http://unix:/home/pi/pyprojects/portfolio_django_1.9/run/gunicorn.sock:/fr/", host: "192.168.1.14", referrer: "http://192.168.1.14/en/"
2016/06/03 19:44:32 [error] 645#0: *1 open() "/home/pi/pyprojects/portfolio_django_1.9/static/500.html" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /fr/ HTTP/1.1", upstream: "http://unix:/home/pi/pyprojects/portfolio_django_1.9/run/gunicorn.sock/fr/", host: "192.168.1.14", referrer: "http://192.168.1.14/en/"
2016/06/03 19:50:20 [error] 640#0: *4 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/fr/"
2016/06/03 19:50:40 [error] 646#0: *1 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/en/"
2016/06/03 21:02:00 [error] 646#0: *3 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/fr/"
2016/06/03 21:18:17 [error] 646#0: *7 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/fr/"
2016/06/03 21:18:29 [error] 646#0: *7 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/fr/"
2016/06/03 21:20:14 [error] 646#0: *28 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/fr/"
2016/06/03 21:20:23 [error] 646#0: *26 directory index of "/home/pi/pyprojects/portfolio_django_1.9/static/" is forbidden, client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/ HTTP/1.1", host: "192.168.1.14"
2016/06/03 21:20:36 [error] 646#0: *28 directory index of "/home/pi/pyprojects/portfolio_django_1.9/static/" is forbidden, client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/ HTTP/1.1", host: "192.168.1.14"
2016/06/03 21:20:36 [error] 646#0: *32 directory index of "/home/pi/pyprojects/portfolio_django_1.9/static/" is forbidden, client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/ HTTP/1.1", host: "192.168.1.14"
2016/06/03 21:20:37 [error] 646#0: *32 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/03 21:20:53 [error] 646#0: *32 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/03 21:20:56 [error] 646#0: *26 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/03 21:22:13 [error] 646#0: *37 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/03 21:29:18 [error] 646#0: *42 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/03 22:18:09 [error] 646#0: *47 open() "/home/pi/pyprojects/portfolio_django_1.9/static/images/favicon.ico" failed (2: No such file or directory), client: 192.168.1.18, server: lefrancjoaquim.com, request: "GET /static/images/favicon.ico HTTP/1.1", host: "192.168.1.14"
2016/06/03 22:18:09 [error] 646#0: *44 open() "/home/pi/pyprojects/portfolio_django_1.9/static/images/favicon.ico" failed (2: No such file or directory), client: 192.168.1.18, server: lefrancjoaquim.com, request: "GET /static/images/favicon.ico HTTP/1.1", host: "192.168.1.14"
2016/06/03 22:18:10 [error] 646#0: *49 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.18, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/05 22:59:44 [error] 650#0: *7 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/06 18:43:32 [error] 641#0: *19 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/"
2016/06/06 18:47:58 [error] 641#0: *25 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/en/"
2016/06/06 18:48:26 [error] 641#0: *23 open() "/home/pi/pyprojects/portfolio_django_1.9/media/images/Capture_décran_2016-05-21_à_16.44.31.png" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /media/images/Capture_d%C3%A9cran_2016-05-21_%C3%A0_16.44.31.png HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/fr/"
2016/06/07 01:58:02 [error] 667#0: *23 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/jquery/jquery.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/home/image/"
2016/06/07 01:58:02 [error] 667#0: *23 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/xregexp/xregexp.min.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/home/image/"
2016/06/07 01:58:06 [error] 667#0: *17 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/jquery/jquery.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/home/categorie/"
2016/06/07 01:58:06 [error] 667#0: *20 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/xregexp/xregexp.min.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/home/categorie/"
2016/06/07 01:58:08 [error] 667#0: *17 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/jquery/jquery.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/auth/group/"
2016/06/07 01:58:08 [error] 667#0: *20 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/xregexp/xregexp.min.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/auth/group/"
2016/06/07 01:58:08 [error] 667#0: *9 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/img/search.svg" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/img/search.svg HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/auth/group/"
2016/06/07 01:58:10 [error] 667#0: *20 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/xregexp/xregexp.min.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/xregexp/xregexp.min.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/auth/user/"
2016/06/07 01:58:10 [error] 667#0: *17 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/js/vendor/jquery/jquery.js" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/auth/user/"
2016/06/07 01:58:11 [error] 667#0: *17 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/img/search.svg" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/img/search.svg HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/auth/user/"
2016/06/07 01:58:11 [error] 667#0: *9 open() "/home/pi/pyprojects/portfolio_django_1.9/static/admin/img/icon-yes.svg" failed (2: No such file or directory), client: 192.168.1.25, server: lefrancjoaquim.com, request: "GET /static/admin/img/icon-yes.svg HTTP/1.1", host: "192.168.1.14", referrer: "http://192.168.1.14/admin/auth/user/"
2016/06/07 14:45:33 [error] 5930#0: *16 directory index of "/home/pi/pyprojects/portfolio_django_1.9/static/" is forbidden, client: 192.168.1.25, server: 192.168.1.14, request: "GET /static/ HTTP/1.1", host: "192.168.1.14"
2016/06/07 16:21:17 [error] 656#0: *32 directory index of "/home/pi/pyprojects/portfolio_django_1.9/media/images/" is forbidden, client: 192.168.1.25, server: raspberry.com, request: "GET /media/images/ HTTP/1.1", host: "192.168.1.14"
2016/06/07 16:21:19 [error] 656#0: *35 directory index of "/home/pi/pyprojects/portfolio_django_1.9/media/images/" is forbidden, client: 192.168.1.25, server: raspberry.com, request: "GET /media/images/ HTTP/1.1", host: "192.168.1.14"
2016/06/07 16:21:19 [error] 656#0: *37 directory index of "/home/pi/pyprojects/portfolio_django_1.9/media/images/" is forbidden, client: 192.168.1.25, server: raspberry.com, request: "GET /media/images/ HTTP/1.1", host: "192.168.1.14"
2016/06/07 16:21:20 [error] 656#0: *32 directory index of "/home/pi/pyprojects/portfolio_django_1.9/media/" is forbidden, client: 192.168.1.25, server: raspberry.com, request: "GET /media/ HTTP/1.1", host: "192.168.1.14"
2016/06/07 16:21:22 [error] 656#0: *37 directory index of "/home/pi/pyprojects/portfolio_django_1.9/media/" is forbidden, client: 192.168.1.25, server: raspberry.com, request: "GET /media/ HTTP/1.1", host: "192.168.1.14"
2016/06/07 16:21:22 [error] 656#0: *38 directory index of "/home/pi/pyprojects/portfolio_django_1.9/media/" is forbidden, client: 192.168.1.25, server: raspberry.com, request: "GET /media/ HTTP/1.1", host: "192.168.1.14"
